package.json

```diff
@@ -67,7 +67,9 @@
         "production": "npm start",
         "unlocalize": "grunt unlocalize",
         "test": "grunt test && grunt build-browser-compressed",
-        "start": "http-server -p 8000 --cors"
+        "server": "http-server -p 8000 --cors",
+        "start": "npm run localize && npm run server && npm run unlocalize",
+        "start-unlocalized": "npm run server"
```

Comment:
this script is unnecessary if it's just running another npm script. I would say remove it and just stick to `npm run server`